NwPluginAPI/Enums/ServerEventType.cs

```diff
@@ -270,13 +278,13 @@ public enum ServerEventType : int
         /// </remarks>
         PlayerInteractScp330 = 31,
 
-        /// <summary>
-        /// Executed when player interacts with shooting target.
-        /// </summary>
-        /// <remarks>
-        /// Parameters: <see cref="IPlayer"/> player.
-        /// </remarks>
-        PlayerInteractShootingTarget = 32,
+		/// <summary>
+		/// Executed when player interacts with shooting target.
+		/// </summary>
+		/// <remarks>
+		/// Parameters: <see cref="IPlayer"/> player, <see cref="ShootingTarget"/> shootingTarget
```

Comment:
Can't it be just `target`? One word names are better than two word names if the meaning is obvious anyway.